cypress/integration/Common.feature

```diff
@@ -0,0 +1,50 @@
+Feature: Test cases for common elements and functionalities
+
+  Scenario Outline: Searching for an address
+    Given User has opened the "landing" page
+    And User clicks on the header search bar
+    And User searches for static balance account
+    And User opens the "<network>" account result
+    Then The account page on "<network>" is opened
+    Examples:
+      | network          |
+      | matic            |
+      | xdai             |
+      | arbitrum-rinkeby |
+      | avalanche-fuji   |
+      | goerli           |
+      | kovan            |
+      | mumbai           |
+      | optimism-kovan   |
+      | rinkeby          |
+      | ropsten          |
+
+  Scenario: Searching for a listed token
+    Given User has opened the "landing" page
+    And User clicks on the header search bar
```

Comment:
Instead of "click on the header search bar", I would write "opens search dialog". Very small thing but it should be a general pattern to write about intent and wanted functionality (opening a search dialog) instead of the implementation (which in this case is _clicking on the header_).

Reply:
Done 👍